jma_customizations.module

```diff
@@ -487,6 +491,47 @@ function jma_customizations_views_pre_render(ViewExecutable &$view) {
     }
     $view->result = $results;
   }
+  if ($view->id() == 'event_location') {
+    foreach ($view->result as $k => $row) {
+      if ($row->_entity) {
+        $loc_block_id = $row->_entity->toArray()['loc_block_id'];
+        if (!empty($loc_block_id[0]['target_id'])) {
+          $sql = "SELECT CONCAT(street_address, ',', city, ',', postal_code, ',', 'Ontario') FROM civicrm_address a INNER JOIN civicrm_loc_block lb ON lb.address_id = a.id WHERE lb.id = " . $loc_block_id[0]['target_id'];
+          $view->result[$k]->_entity->set('summary', str_replace(',', "\n", \CRM_Core_DAO::singleValueQuery($sql)));
+        }
+      }
+      else {
+        $id = $row->id;
+        $contact_ent = \Drupal::entityTypeManager()->getStorage('civicrm_contact')->load($id);
+        if (!empty($view->result[0]->_entity)) {
+          $row->_entity = clone $view->result[0]->_entity;
+        }
+        else {
+          $row->_entity = \Drupal::entityTypeManager()->getStorage('civicrm_event')->load(11);
```

Comment:
@monishdeb any reason why we pick the id 11?

Reply:
@seamuslee001 as it needs an empty skeletal event entity, so in case there aren't any event row present in the view result use 11 which is a reserved event template id.